e2e/steps/common.mjs

```diff
@@ -67,32 +84,32 @@ Given('I wait for {string}', async (timeout) => {
   await new Promise((res) => setTimeout(res, +time * unitMultiplier[unitKey]));
 });
 
-Then('I should see {string}', async function (textContent) {
+Then('I should see {string}', async (textContent) => {
   await expect(fixture.page.getByText(textContent, { exact: true })).toBeVisible();
 });
 
-Then('I should possibly see {string}', async function (textContent) {
+Then('I should possibly see {string}', async (textContent) => {
```

Comment:
I think we need to use the function here

Reply:
Prettier removes it on save. It wasn't a manual change

Reply:
https://github.com/cucumber/cucumber-js/issues/2004

Reply:
While I can see that issue, there was never a PR for it and so far, we don't use the `this`/`world` context. Furthermore, any other change to that file by anyone else would still format it on save except Prettier is disabled locally

Reply:
This is probably some local linting you have 🤔.
It doesnt format it for me, i dont think prettier nor eslint settings formats this since it basically mean two different things 🤔. If the code produced identical results it would make sense that these tools formatted it, but not when they mean different things.

Another motivation from me to keep previous syntax (except for being able to add world) is that it will cause merge conflict with my PR

Reply:
Fixed